### PR TITLE
Remove HQPillow and subclasses

### PR DESCRIPTION
**Pull request by snopoke**
@czue

@calellowitz 

review by commit :fish:

### CODE REVIEW

File: corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py [hunk 1]
```diff
@@ -345,27 +344,7 @@ def finish_saving(self):
...
-                break
-            except Exception as ex:
-                retries += 1
-                retry_time = (datetime.utcnow() - bulk_start).seconds + retries * RETRY_TIME_DELAY_FACTOR
-                self.log("\t%s: Exception sending slice %d:%d, %s, retrying in %s seconds" % (datetime.utcnow().isoformat(), start, end, ex, retry_time))
-                time.sleep(retry_time)
-                self.log("\t%s: Retrying again %d:%d..." % (datetime.utcnow().isoformat(), start, end))
-                # reset timestamp when looping again
-                bulk_start = datetime.utcnow()
+        raise NotImplementedError
```
Comment (snopoke) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py]: This whole file can go pretty soon so not putting much effort in here

File: corehq/ex-submodules/pillowtop/listener.py [hunk 2]
```diff
@@ -380,182 +380,6 @@ def send_to_elasticsearch(index, doc_type, doc_id, es_getter, name, data=None, r
...
-        class name and the hashed name representation.
-        """
-        return self.get_legacy_name()
-
-    @classmethod
-    def get_legacy_name(cls):
-        return "%s.%s.%s.%s" % (
-            cls.__module__, cls.__name__, cls.get_unique_id(), get_machine_id())
-
-
```
Comment (czue) [inline corehq/ex-submodules/pillowtop/listener.py]: :skull:

**Comment by czue** [comment]
looks good but not mergeable. thanks for finishing what i started! is amazing to finally see this go

**Comment by snopoke** [comment]
:white_check_mark: (note that I added a new commit after merging master: https://github.com/dimagi/commcare-hq/pull/12526/commits/b49e78eaba13001122d6827bddbe1c5230efe624)

**Pull request merged**